### PR TITLE
Fix: Ensure there's a "v" prefix in markers version

### PR DESCRIPTION
**Pull request by azylinski**
Ensure there's a consistent versioning (with "v" prefix) in build markers.

Constraint:
 - the version param MAY have a "v" prefix, e.g. `1.25.0` or `v1.24.4-rc.0`
 - fileName MUST NOT have a "v" prefix, e.g.: `latest-green-1.23.txt`, `latest-1.22.txt`
 - content (marker) MUST have "v" prefix, e.g.: `v1.22.0-alpha.3`

Ref: https://github.com/kubernetes-sigs/kubetest2/pull/185#discussion_r915831056

### CODE REVIEW

File: kubetest2-gke/deployer/utils/util.go [hunk 1]
```diff
@@ -27,26 +27,32 @@ import (
...
-		minor := m[1]
-		fName = fmt.Sprintf("%s-%s.txt", markerPrefix, minor)
-	}
-	pushCmd := fmt.Sprintf("gsutil -h 'Content-Type:text/plain' cp - %s/%s", stageLocation, fName)
+	destinationURL := stageLocation + "/" + fileName(version, markerPrefix)
+	pushCmd := fmt.Sprintf("gsutil -h 'Content-Type:text/plain' cp - %s", destinationURL)
 	cmd := exec.RawCommand(pushCmd)
+	if !strings.HasPrefix(version, "v") {
+		version = "v" + version
+	}
```
Comment (azylinski) [inline kubetest2-gke/deployer/utils/util.go:40]: Same as: https://github.com/kubernetes-sigs/kubetest2/blob/master/kubetest2-gke/deployer/build/gke_make.go#L107-L109

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve

**Comment by BenTheElder** [comment]
@mborsz the bot comment means you're not a kubernetes-sigs member, you can correct that pretty easily in https://github.com/kubernetes/org since you're already a member of github.com/kubernetes org, FWIW

File: kubetest2-gke/deployer/utils/util.go [hunk 2]
```diff
@@ -27,26 +27,32 @@ import (
...
-	} else {
-		minor := m[1]
-		fName = fmt.Sprintf("%s-%s.txt", markerPrefix, minor)
-	}
-	pushCmd := fmt.Sprintf("gsutil -h 'Content-Type:text/plain' cp - %s/%s", stageLocation, fName)
+	destinationURL := stageLocation + "/" + fileName(version, markerPrefix)
+	pushCmd := fmt.Sprintf("gsutil -h 'Content-Type:text/plain' cp - %s", destinationURL)
 	cmd := exec.RawCommand(pushCmd)
+	if !strings.HasPrefix(version, "v") {
+		version = "v" + version
```
Comment (liggitt) [inline kubetest2-gke/deployer/utils/util.go:39]: adding a `v` prefix breaks existing consumers of this file, please revert this change
